frontend/src/apiMocks/handlers.ts

```diff
@@ -285,6 +287,94 @@ export function getHandlers(
     ownAddresses.splice(index, 1);
     return res(ctx.status(200));
   });
+
+  addGetHandler("/api/v1/realphone/", (req, res, ctx) => {
+    const mockId = getMockId(req);
+    if (mockId === null) {
+      return res(ctx.status(400));
+    }
+
+    return res(ctx.status(200), ctx.json(mockedRealphones[mockId]));
+  });
+  addPostHandler("/api/v1/realphone/", (req, res, ctx) => {
```

Comment:
Both you and @maxxcrawford - would this code be a lot simpler if I removed the ability to `POST` the verification code? So the only kind of request that would be a "verification" request is a `PATCH`?

Reply:
I think indeed that having just one way to verify phone numbers would be simpler, _but_ IIRC Maxx's current code is using `POST` to verify. It's not necessarily much work to change, but given that there's already quite a bit on his plate, maybe we should keep that as a future enhancement.

Reply:
Heh. I'm using [them](https://github.com/mozilla/fx-private-relay/blob/start-phone-api-react/frontend/src/hooks/api/phone.ts#L140) [both](https://github.com/mozilla/fx-private-relay/blob/start-phone-api-react/frontend/src/hooks/api/phone.ts#L124). 😅